src/service/fixed-layer.js

```diff
@@ -269,9 +269,14 @@ export class FixedLayer {
 
         // 1. Set all style top to `auto` and calculate the auto-offset.
         for (let i = 0; i < elements.length; i++) {
-          const fe = elements[i];
-          oldTops.push(getStyle(fe.element, 'top'));
-          setStyle(fe.element, 'top', 'auto');
+          const el = elements[i].element;
+          const style = computedStyle(this.ampdoc.win, el);
+          styles.push(style);
+          if (endsWith(style.position || '', 'sticky')) {
+            setStyle(el, 'position', 'fixed');
```

Comment:
What does this attempt to do? Generally, `position: sticky` should be completely ignored for transfer layer. It's simply impossible to transfer it.